x-pack/platform/plugins/shared/aiops/public/embeddables/log_rate_analysis/log_rate_analysis_config_input.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { AiopsPluginStartDeps } from '../../types';
+import { LogRateAnalysisEmbeddableInitializer } from './log_rate_analysis_embeddable_initializer';
+import type { LogRateAnalysisComponentApi, LogRateAnalysisEmbeddableState } from './types';
+
+export function EmbeddableLogRateAnalysisUserInput({
+  pluginStart,
+  isNewPanel,
+  logRateAnalysisControlsApi,
+  onCancel,
+  onConfirm,
+  deletePanel,
+  initialState,
+}: {
+  pluginStart: AiopsPluginStartDeps;
+  isNewPanel: boolean;
```

Comment:
nit: you can make it optional and not pass it where you pass `false` instead
```suggestion
  isNewPanel?: boolean;
```

Reply:
I decided not to make `isNewPanel` optional, because `LogRateAnalysisEmbeddableInitializer` expects it to be a boolean, and passing undefined causes a TypeScript error.